apps/chat/src/components/Common/ApplicationWizard/ApplicationWizardHeader.tsx

```diff
@@ -0,0 +1,50 @@
+import { IconX } from '@tabler/icons-react';
+import { FC } from 'react';
+
+import { useTranslation } from 'next-i18next';
+
+import { ApplicationType } from '@/src/types/applications';
+import { Translation } from '@/src/types/translation';
+
+const getTitle = (type: ApplicationType, isEdit: boolean) => {
+  switch (type) {
+    case ApplicationType.QUICK_APP:
+      return isEdit ? 'Edit quick app' : 'Add quick app';
+    case ApplicationType.CODE_APP:
+      return isEdit ? 'Edit code app' : 'Add code app';
+    case ApplicationType.CUSTOM_APP:
+    default:
+      return isEdit ? 'Edit application' : 'Add application';
+  }
```

Comment:
Add values into enum ApplicationType with label of this types
```
export enum ApplicationType {
  CUSTOM_APP = 'application',
  QUICK_APP = 'quick app',
  CODE_APP = 'code app',
}
```
and just use it on UI
```suggestion
const getTitle = (type: ApplicationType, isEdit?: boolean) => `${isEdit ? 'Edit' : 'Add'} Edit ${type}`;
```